### ISSUE TITLE
feature_importance

### ISSUE THREAD

**Issue by AVPokrovsky** [JoaquinAmatRodrigo/skforecast#390]
Hi, I am trying to check feature importance. Some numbers are positive and some are negative. How to understand it? Which ones should be considered as more important?

https://joaquinamatrodrigo.github.io/skforecast/0.7.0/api/ForecasterMultiVariate.html#skforecast.ForecasterAutoregMultiVariate.ForecasterAutoregMultiVariate.ForecasterAutoregMultiVariate.get_feature_importance

[screen.docx](https://github.com/JoaquinAmatRodrigo/skforecast/files/11161937/screen.docx)

**Comment by JoaquinAmatRodrigo** [comment]
Hi @AVPokrovsky, thanks for using skforecast.

If you get negative values for the importance of the characteristic, I assume you are using a linear regressor. If this is the case, the characteristic importance values are the regression coefficients (coeff_). Here you can find some good explanations on how to interpret them:

-  https://machinelearningmastery.com/calculate-feature-importance-with-python/
-  https://inria.github.io/scikit-learn-mooc/python_scripts/dev_features_importance.html#linear-model-inspection

In summary, if you have scaled your features, the absolute value of the regression coefficients indicates the importance of each feature **in the model**. The sign indicates the direction of the influence on the target variable.

**Comment by AVPokrovsky** [comment]
Thanks
